lib/plugins/TaskRunner/LAMPApp.js

```diff
@@ -12,8 +12,15 @@ class LAMPApp extends Script {
    *   @param {object} container - An instantiated and configured Container object.
    *   @param {object} options - A hash of configuration options specific to this task.
    *   @param {string} [options.database] - The file name of the database to import if specified. Note that this database *must be added to the assets array separately*.
-   *   @param {string} [options.databaseName] - The name of the database to use.
-   *   @param {string} [options.databaseUser] - The username of the database to use.
+   *   @param {string} [options.databaseName] - The name of the database to use. Defaults to 'lampdb'.
+   *   @param {string} [options.databaseUser] - The username of the database to use. Defaults to 'root'.
+   *   @param {string} [options.databasePass] - The password for the database username. Defaults to 'strongpassword'.
+   *     - Note not recommended to pass credentials in options.
+   *       Instead, include a credentials.sh assets file https://docs.probo.ci/build/assets/ which contains e.g.:
+   *          #!bin/bash
+   *          DATABASE_NAME='foo'
+   *          DATABASE_USER='bar'
+   *          DATABASE_PASS='baz'
```

Comment:
The password here doesn't seem that important since it's a temporary machine and the current password is 'strongpassword'.  Could we drop the code that sets a password on a new mysql user and default to an empty string or 'strongpassword'?

Then we just need to document it.

Reply:
I put in line 17 comment that the default is strongpassword (we'll need to update docs as well) and then on line 48 make that the default, so I think we're ok.

I think the issue with allowing people the option to specify a password (as mentioned in https://github.com/ProboCI/probo/issues/65) was really born from the fact that we never documented what the default db name, user and password were.

I guess the ability to customize the password is really not necessary but since we're letting them customize the db name and user (equally unnecessary) it  makes sense to have all three.